packages/styles/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,112 @@
+@use "../tokens" as *;
+@use "../functions" as *;
+@import "../mixins";
+
+/**
+This mixin is used in _richtext.scss component to style blockquotes.
+It could eventually be used for a standalone blockquote component.
+**/
+@mixin blockquote {
```

Comment:
If we are implementing the new styles I think it should be compliant to the specification we defined in #999

in this case, we should remove all SCSS variables (excluding `dataurlicon` #1189 ), define the blockquote variables, and reference them

Reply:
Hey @GGKapanadze, we definitely should, I've just created issue for that https://github.com/international-labour-organization/designsystem/issues/1240